### PR TITLE
Added issue template

### PR DESCRIPTION
**Pull request by Aman-Codes**
@priyansh19  Please review my pull request and do let me know if any changes are required.
Added bug_report_template.md and feature_request_template.md.
Closes: #230 
### Bug report
bug report

### Feature request
feature request

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report_template.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 16
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240&bid=17122298)

File: .github/ISSUE_TEMPLATE/bug_report_template.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/feature_request_template.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/feature_request_template.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 1 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/feature_request_template.md (hunk 3 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/feature_request_template.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 1 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/bug_report_template.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 1 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 2 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/bug_report_template.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 4 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/feature_request_template.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/feature_request_template.md]: Codacy Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 4 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 2 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 4 above)
Comment (rpotter12) [inline .github/ISSUE_TEMPLATE/bug_report_template.md]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5222240)

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 1 above)
Comment (priyansh19) [reply .github/ISSUE_TEMPLATE/bug_report_template.md]: you don't need to specify examples

File: .github/ISSUE_TEMPLATE/bug_report_template.md (hunk 2 above)
Comment (priyansh19) [reply .github/ISSUE_TEMPLATE/bug_report_template.md]: No Examples needed.

Comment (priyansh19) [reply .github/ISSUE_TEMPLATE/bug_report_template.md]: dont leave extra spaces it gives indentation error

File: .github/ISSUE_TEMPLATE/feature_request_template.md (hunk 3 above)
Comment (priyansh19) [reply .github/ISSUE_TEMPLATE/feature_request_template.md]: don't use [...] extra trailing or reference links

**Comment by priyansh19** [comment]
they are not as per their naming conventions hence may e they won't work properly

**Comment by priyansh19** [comment]
Please check everything before making any Pull Request

**Comment by priyansh19** [comment]
If you still have any problem, please discuss it in the slack channel of the project

**Pull request merged**